dorado/scheduling/scripts/main.py

```diff
@@ -54,10 +54,19 @@ def parser():
     group.add_argument(
         '--skygrid-step', type=u.Quantity, default='0.0011 sr',
         help='Sky grid resolution (any solid angle units')
+    group.add_argument(
+        '--number_of_orbits', type=int, default=1,
```

Comment:
Please remove this argument, because it is degenerate with the new `--duration` argument, which understands orbits.

```
--duration="1.5 orbit"
--duration="90 min"
--duration="1 hour"
```